google-cloud-core/lib/google/cloud/config.rb

```diff
@@ -458,8 +460,8 @@ def self.deferred &block
       def method_missing name, *args
         name_str = name.to_s
         super unless name_str =~ /^[a-zA-Z]\w*=?$/
-        if name_str.chomp! "="
-          self[name_str] = args.first
+        if name_str.end_with?("=")
+          self[name_str[0..-2]] = args.first
```

Comment:
FWIW, I find this easier to read:

```suggestion
          self[name_str[0...-1]] = args.first
```